codecov_auth/tests/test_admin.py

```diff
@@ -190,25 +185,6 @@ def test_inline_orgwide_add_token_permission_no_token_user_not_in_enterprise_clo
         inline_instance = inlines[0]
         assert inline_instance.has_add_permission(request, owner_in_cloud_plan) == True
 
-    def test_inline_owner_profile_display(self):
-        owner = OwnerFactory()
-        request_url = reverse(f"admin:codecov_auth_owner_change", args=[owner.ownerid])
-        request = RequestFactory().get(request_url)
-        request.user = self.staff_user
-        inlines = self.owner_admin.get_inline_instances(request, owner)
-        assert isinstance(inlines[1], OwnerProfileInline)
-
-    def test_inline_owner_profile_permissions(self):
-        owner = OwnerFactory(name="test-owner", service="github")
-        owner.save()
-        request_url = reverse(f"admin:codecov_auth_owner_change", args=[owner.ownerid])
-        request = RequestFactory().get(request_url)
-        request.user = self.staff_user
-        inlines = self.owner_admin.get_inline_instances(request, owner)
-        inline_instance = inlines[1]
-        assert inline_instance.has_delete_permission(request, owner) == False
-        assert inline_instance.has_change_permission(request, owner) == True
```

Comment:
Did you confirm this logic still works in the django admin for the User model? People using the django admin have come with issues in the past where they are changing other fields in an Owner/User model, and they can't save them because some fields (in this case the terms fields) are required. Wanting to confirm if you can go edit a User entry and have the terms fields not required to save the change.

Reply:
Interesting! It certainly does error in admin page. (When the fields are null it errors, not when there is existing value.)

Reply:
Fixed. See [codecov_auth/models.py](https://github.com/codecov/codecov-api/pull/100/files#diff-e2e21324fdebb10e0effbcbc3cc096e121165c4476b1ff8428bb3a51b34ef14b)

Reply:
peeerfect, thanks!